src/components/Menu.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div :class="bem()" v-if="show">
+    <slot name="header"></slot>
+
+    <div
+      v-for="(facet, key) in facetValues"
+      :key="key"
+      :class="facet.isRefined ? bem('item', 'active') : bem('item')"
+    >
+      <a
+        href="#"
+        :class="bem('link')"
+        @click.prevent="handleClick(facet.path)"
+      >
+        {{facet.name}}
+        <span :class="bem('count')">{{facet.count}}</span>
+      </a>
+    </div>
+
+    <slot name="footer"></slot>
+  </div>
+</template>
+
+<script>import algoliaComponent from '../component';
+  import { FACET_TREE } from '../store';
+
+  export default {
+    mixins: [algoliaComponent],
+
+    props: {
+      attribute: {
+        type: String,
+        required: true,
+      },
+      limit: {
+        type: Number,
+        default: 10,
+      },
+      sortBy: {
+        default() {
+          return ['isRefined:desc', 'count:desc', 'name:asc'];
+        },
+      },
+    },
+
+    computed: {
+      facetValues() {
+        const { data = [] } = this.searchStore.getFacetValues(
+          this.attribute,
+          this.sortBy
+        );
+
+        return data;
+      },
+      show() {
+        return this.facetValues.length > 0;
+      },
+    },
+
+    methods: {
+      handleClick(path) {
+        this.searchStore.toggleFacetRefinement(this.attribute, path);
+      },
+    },
+
+    data() {
+      return {
+        blockClassName: 'ais-menu',
+      };
+    },
+
+    created() {
+      this.searchStore.setMaxValuesPerFacet(this.limit);
```

Comment:
Is this how we do it in other IS? Because this will impact all other components as well.
It think we should remove the limit from this component maybe?

Reply:
Yes this is done like that, we will fix the `maxValuesPerFacet` to the maximum known limit from widgets.

Reply:
Ok, then it needs proper documentation because this can be somewhat misleading.